### DISCUSSION TITLE
[1264970]The Conquest of Go {仅本体}

### DISCUSSION THREAD

**Discussion by logicircle** [pjy612/SteamManifestCache#824] [Share]
### Steam 游戏ID

1264970

### 密钥列表

```hash
1264971;895e7b54910685eab2f85579edfd824aa1f0e871684f9893123fba0ac740d11b
```

### 清单附件

[1264971_2584296244069743183.zip](https://github.com/user-attachments/files/17011741/1264971_2584296244069743183.zip)

**Comment by pjy612** [comment]
不看邮箱的吗？

**Reply by logicircle** [reply]
不好意思，没有看到邮件，刚检查
